### PR TITLE
Changed index file, added favicons and fixed manifest

### PR DESCRIPTION
**Pull request by Toughee**
Types of changes
----------------

-  Bug fix (non-breaking change which fixes an issue)

Description
-----------

* Changed index file to support multiple favicon files. 
* Also updated title for index.html title into: "Request Yo Racks" instead of my app.
* Updated manifest file to show "Request Yo Racks" instead of Polymer app. 
* Removed duplicate entries for logos inside manifest file 
* Added favicon support for Apple, Android and Microsoft.
* Main favicon updated to Request Yo Racks logo.
* Updated tagline for Request Yo Racks
* Updated "What to do next" section and added a Caution message before Preview section

Motivation and Context
----------------------

Fixes multiple minor issues that Request Yo Racks had. Also updates tagline and add Caution message to users

How Has This Been Tested?
-------------------------
 
Changes don't tamper with any code

Checklist:
----------

Fixes request-yo-racks/web#88
Fixes request-yo-racks/web#89
Fixes request-yo-racks/web#102
Fixes request-yo-racks/web#104

### CODE REVIEW

**Comment by Toughee** [comment]
Review @rgreinho

File: manifest.json [hunk 1]
```diff
@@ -1,20 +1,46 @@
...
-      "src": "images/manifest/icon-512x512.png",
-      "sizes": "512x512",
-      "type": "image/png"
-    }
-  ]
+  {
+   "src": "images/manifest/android-icon-36x36.png",
+   "sizes": "36x36",
+   "type": "image/png",
+   "density": "0.75"
```
Comment (rgreinho) [inline manifest.json]: What is density?

File: manifest.json [hunk 2]
```diff
@@ -1,20 +1,46 @@
...
-      "type": "image/png"
-    },
-    {
-      "src": "images/manifest/icon-512x512.png",
-      "sizes": "512x512",
-      "type": "image/png"
-    }
-  ]
+  {
+   "src": "images/manifest/android-icon-36x36.png",
```
Comment (rgreinho) [inline manifest.json]: There is no need to duplicate the icons. You can directly use the ones we have in the `images/logos` folder.

Comment (Toughee) [reply manifest.json]: Okay. I'll go ahead and change that now

File: manifest.json (hunk 1 above)
Comment (Toughee) [reply manifest.json]: It's related to the image resolution. Since you already had icons before the PR, I'll remove that section.

**Comment by Toughee** [comment]
Making a new pull request now with changes requested

File: index.html [hunk 3]
```diff
@@ -16,44 +16,71 @@
...
-  <!-- See https://goo.gl/OOhYW5 -->
+  <link rel="icon" href="images/manifest/favicon.ico">
+  <link rel="icon" type="image/png" sizes="16x16" href="images/manifest/favicon-16x16.png">  
+  <link rel="icon" type="image/png" sizes="32x32" href="images/manifest/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="96x96" href="images/manifest/favicon-96x96.png">
   <link rel="manifest" href="manifest.json">
+  <meta name="theme-color" content="#ffffff">
+
+  <!-- Android Icons -->
+  <link rel="apple-touch-icon" sizes="36x36" href="images/manifest/android-icon-36x36.png">
```
Comment (rgreinho) [inline index.html]: There is no need to have platform specific icons for android/apple/ms. They should all point to the `images/logos` directory, and you should be able to remove the images from `images/manifest`. 

If some icon sizes are missing you may add them in `images/logos`. Check with @2echoi if you need further help with the logos.

File: src/my-view1.html [hunk 4]
```diff
@@ -193,7 +193,9 @@
     <div class="content">
       <div class="description">
       <h2 class="intro-description center color-secondary">
-        Create an email to request a free bike corral from the City of Austin
+        <strong>Make your voice heard!</strong>
```
Comment (rgreinho) [inline src/my-view1.html]: This tagline sounds a bit too much like activists. Although I like the idea, I believe we should stick to a simple tagline which conveys the message that the bike rack request process is now easy.

File: src/my-view1.html [hunk 5]
```diff
@@ -193,7 +193,9 @@
     <div class="content">
       <div class="description">
       <h2 class="intro-description center color-secondary">
-        Create an email to request a free bike corral from the City of Austin
+        <strong>Make your voice heard!</strong> 
+        <br>
+        Request Yo Racks is a free, responsive web application that helps you to request bike corrals to be installed directly to the City of Austin via email.
```
Comment (rgreinho) [inline src/my-view1.html]: I like the idea of the elevator speech. However it should not be placed here. It fills all the space on the image and breaks the mobile version of the web app.

Desktop version:
screen shot 2018-06-28 at 10 28 10 am

Mobile version:
tagline - mobile

File: src/my-view1.html [hunk 6]
```diff
@@ -246,6 +248,16 @@ <h2 class="section-title bg-secondary center">Select a location</h2>
       <section id="preview">
         <ryr-email-templatizer templatevars="{{apivars}}" htmlrenderedtemplate="{{buffer}}" markdownrenderedtemplate="{{md}}"></ryr-email-templatizer>
         <h2 class="section-title bg-primary center">Preview</h2>
+        
+        <div class="warning-box">
+          <iron-icon icon="vaadin:warning"></iron-icon>
+          <p class="justify">
+            Make sure the business you chose supports your request. Without it, the request will be not be processed. You must also go talk to the businesses supporting their request in order for the bike rack to be installed.
```
Comment (rgreinho) [inline src/my-view1.html]: You can remove the 3rd sentence, it does not really add value. Besides, requesters can contact the businesses by other ways than by going to talk to the manager (mail, email, phone, etc.).

File: src/my-view1.html [hunk 7]
```diff
@@ -246,6 +248,16 @@ <h2 class="section-title bg-secondary center">Select a location</h2>
...
+            <br>
+            <strong>There is no guarantee that the bike rack will be installed.</strong>
```
Comment (rgreinho) [inline src/my-view1.html]: I like this very much! Please check with @2echoi and @UX-Chris to see how we can make it fit the new design.

File: index.html [hunk 8]
```diff
@@ -16,44 +16,71 @@
   <meta name="generator" content="Polymer Starter Kit">
   <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
 
-  <title>My App</title>
-  <meta name="description" content="My App description">
+  <title>Request Yo Racks</title>
+  <meta name="description" content="Request Yo Rack">
```
Comment (rgreinho) [inline index.html]: The description should be "Request a bike rack in 3 steps".

File: src/my-view1.html (hunk 4 above)
Comment (Toughee) [reply src/my-view1.html]: Okay. I'll update it now

File: src/my-view1.html (hunk 6 above)
Comment (Toughee) [reply src/my-view1.html]: Removing the 3rd sentence now

**Pull request closed without merging**